src/app/components/passwordBadge.component.ts

```diff
@@ -0,0 +1,15 @@
+import {InputValidator} from '../components'
+
+export class PasswordBadge{
+    public static create(passwordState : string) : HTMLSpanElement{
+        const badge = document.createElement('span');
+        badge.id = 'passwordBadge';
+        badge.textContent = passwordState;
+        return badge;
+    }
+
+    public static check(passwordText: string) : string{
```

Comment:
Please, check usage and remove if we don't need it

Reply:
done